vagrantfile

```diff
@@ -0,0 +1,44 @@
+# -*- mode: python -*- 
```

Comment:
maybe better rename the file to `Vagrantfile` (uppercase) for better consistency (comparison to old version/usage in other TIB boxes)